iOS/MMIWG2S/Views/ARFaceViewerUI.swift

```diff
@@ -67,13 +67,45 @@ class ARFaceViewerUI: UIView {
         self.redButton = redButton
     }
     
-    @objc private func shareButtonTapped() {
-        guard let imageToShare = delegate?.generateShareImage() else { return }
-        
-        // Use this to upload to the database. This will likely move :)
-//        let scaledDownImage = imageToShare.compress(to: 1000)
+    private func uploadImage(_ compressedImage: UIImage, name: String, email: String) {
+        guard let nameData = name.data(using: .utf8),
+              let emailData = email.data(using: .utf8),
+              let verifiedData = "1".data(using: .utf8)
+        else { return }
         
-        let activityController = UIActivityViewController(activityItems: [imageToShare], applicationActivities: nil)
+        guard let imageData = compressedImage.pngData() else {
+            log.error("Unable to get image data. Posting user metadata without image")
+            Network.post(
+                url: "https://mmiw.mehequanna.com/submissions",
+                data: [
+                    "name" : ("text", nameData),
+                    "email" : ("text", emailData),
+                    "verified" : ("text", verifiedData)
+                ],
+                contentType: .multipartForm,
+                completionHandler: { error, response in
+                    log.info(String(data: response!, encoding: .utf8)!)
```

Comment:
Can we wrap this in an if let or guard statement, so it doesn't crash the app for some reason